index.html

```diff
@@ -0,0 +1,125 @@
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <main>
+        <div class="hero">
+          <div class="card1">
+          <h1 class="home_title">TRAFFIC & CONVERSION SUMMIT 2022 </h1>
+          <h2 class="home_title2">The congress of online advertising!</h2>
+          <p class="description"> More than 10 hours of training divided into three days in which you will learn from active professionals knowledge, tips, and strategies of online advertising and digital strategy. Whether you're starting from scratch or already have experience,.</p>
+          <div class="dateInfo"> <font>2022.09.13 (Tue) ~ 15 (Thu)</font> </div>
+          <div class="locate">
+            <font>@ Sports Center San Miguel de Sema Boyaca - Colombia</font>
+          </div>
+        </div>
+      </div>
+      <section class="mainProgram">
+        <h1 class="program" id="mainProgram">Main Program</h1>
+        <div class="guideBar"> </div>
+        <div class="programDetail">
+        <div class="programList">
+          <div class="programIcon"><img src="./src/images/program_icon_01.png" alt="conference"></div>
+          <div class="programTitle"><p>conference</p></div>
+          <div class="programDescription"><p>The Simple Two-step Marketing System You Can Copy</P></div>
+        </div>
+        <div class="programList">
+          <div class="programIcon"><img src="./src/images/program_icon_02.png" alt="conference"></div>
+          <div class="programTitle"><p>exhibition</p></div>
+          <div class="programDescription"><p>You will know the keys to growing professionally.</P></div>
+        </div>
+        <div class="programList">
+          <div class="programIcon"><img src="./src/images/program_icon_03.png" alt="conference"></div>
+          <div class="programTitle"><p>forum</p></div>
+          <div class="programDescription"><p>You will learn to invest in online advertising .</P></div>
+        </div>
+        <div class="programList">
+          <div class="programIcon"><img src="./src/images/program_icon_04.png" alt="conference"></div>
+          <div class="programTitle"><p>workshop</p></div>
+          <div class="programDescription"><p>Understanding The New Mechanics Of Digital Marketing</P></div>
+        </div>
+        <div class="programList">
+          <div class="programIcon"><img src="./src/images/program_icon_05.png" alt="conference"></div>
+          <div class="programTitle"><p>Party</p></div>
+          <div class="programDescription"><p>How Businesses are Using Affiliate Partnerships</P></div>
+        </div>
+      </div>
+     </section>
+      <!--  Start  dinamic Content JavaScript object -->
+
+      <section>
+          <p class="speakers_title" id="speakers">Featured Speakers</p>
+          <div class="guideBar"></div>
+      </section>
+ <section class="speakerSection" id="speaker">
+  </section>
+
+  <section class="aboutSectionPartner">
+      <div class="partners ">
```

Comment:
- [ ] I noticed your partners section in mobile view is overflowing, kindly fix this, please.
![2](https://user-images.githubusercontent.com/94551484/190778426-271a82a0-d6a7-4157-a0f1-9143677395fd.png)